gdnative-core/src/byte_array.rs

```diff
@@ -84,6 +88,22 @@ impl ByteArray {
         unsafe { (get_api().godot_pool_byte_array_size)(&self.0) }
     }
 
+    pub fn read<'a>(&'a self) -> Read<'a> {
+        unsafe {
+            MaybeUnaligned::new(ReadGuard::new(self.sys()))
+                .try_into_aligned()
+                .expect("Pool array access should be aligned. This indicate a bug in Godot")
```

Comment:
```suggestion
                .expect("Pool array access should be aligned. This indicates a bug in Godot")
```

This seems to be the same string in all the accessor functions, I will not spam you with the same suggestion all over again :smile:

Reply:
Done!